schema/xml/metaschema.xsd

```diff
@@ -1227,9 +1255,14 @@
         </xs:element>
         <xs:element name="expect" type="ExpectConstraintType">
           <xs:annotation>
-            <xs:documentation>Checks that the specified test returns true in this evaluation context.</xs:documentation>
+            <xs:documentation>Checks that the specified test returns true in this evaluation context. If false, a constraint violation message is returned.</xs:documentation>
           </xs:annotation>
         </xs:element>
+        <xs:element name="report" type="ReportConstraintType">
```

Comment:
I don't see the equivalent of this modeled in the Metaschema module related to flags. Is this intended?

Reply:
No, that wasn't intentional. Is this what I was missing?

https://github.com/metaschema-framework/metaschema/pull/61/commits/8674fe52584e0d9cbc22771569ea1c8ea8525ab4

Reply:
Yep. Thanks. We can merge once this is marked ready.